app/lib/hooks/useVideoConf.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+
+import { callJitsi } from '../methods';
+import { compareServerVersion, showErrorAlert } from '../methods/helpers';
+import { ISubscription, SubscriptionType } from '../../definitions';
+import { useAppSelector } from './useAppSelector';
+import { videoConfStartAndJoin } from '../methods/videoConf';
+import { Services } from '../services';
+import { useActionSheet } from '../../containers/ActionSheet';
+import { useSnaps } from './useSnaps';
+import StartACallActionSheet from '../../containers/UIKit/VideoConferenceBlock/components/StartACallActionSheet';
+import { getSubscriptionByRoomId } from '../database/services/Subscription';
+import { getUserSelector } from '../../selectors/login';
+import i18n from '../../i18n';
+
+const availabilityErrors = {
+	NOT_CONFIGURED: 'video-conf-provider-not-configured',
+	NOT_ACTIVE: 'no-active-video-conf-provider',
+	NO_APP: 'no-videoconf-provider-app'
+} as const;
+
+const handleErrors = (isAdmin: boolean, error: typeof availabilityErrors[keyof typeof availabilityErrors]) => {
+	if (isAdmin) return showErrorAlert(i18n.t(`admin-${error}-body`), i18n.t(`admin-${error}-header`));
+	return showErrorAlert(i18n.t(`${error}-body`), i18n.t(`${error}-header`));
+};
+
+export const useVideoConf = (rid: string): { showInitCallActionSheet: () => Promise<void>; showCallOption: boolean } => {
+	const [showCallOption, setShowCallOption] = useState(false);
+
+	const serverVersion = useAppSelector(state => state.server.version);
+	const jitsiEnabled = useAppSelector(state => state.settings.Jitsi_Enabled);
+	const jitsiEnableTeams = useAppSelector(state => state.settings.Jitsi_Enable_Teams);
+	const jitsiEnableChannels = useAppSelector(state => state.settings.Jitsi_Enable_Channels);
+	const user = useAppSelector(state => getUserSelector(state));
+
+	const greaterThanFive = compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '5.0.0');
```

Comment:
Optional, but the naming doesn't tell the whole story imo.
`greater than five` isn't tied to server so it might cause confusion for someone who never touched the code. Like, five what? It's also incorrect, since the logic is actually `greaterThanOrEqualTo` instead of `greaterThan`.
Thinking of `isServer5OrNewer` or something like that.

We could also come up with a different solution for the future, to make these kinds of tests more clear.